src/ebur128.rs

```diff
@@ -719,6 +779,13 @@ impl EbuR128 {
     }
 
     /// Get global integrated loudness in LUFS across multiple instances.
+    ///
+    /// This can be used to allow parallel iteration of long signals, assuming some care is taken:
+    ///  1. Divide input-signal up in "chunks" of even 100ms samples. Make chunks overlap by 400ms, for example (0-10s, 9.6-20s, 19.6-30s, ...)
+    ///  2. For each chunk, create a new instance of EbuR128, then in parallel:
+    ///     1. Feed the first 100ms of the chunk through seed_frames_* function. This is sufficient to make filter-states in each instance what they would have been if a single analyzer would have reached this point.
+    ///     2. Process the remaining samples of each chunk through the analyzer
+    ///  3. Call loudness_global_multiple over all the chunks to get the global loudness
```

Comment:
```suggestion
    ///  3. Call [`loudness_global_multiple`] over all the chunks to get the global loudness
```

Does this link and the others in `seed_samples()` work like this btw?

Reply:
Will have to check

Reply:
It does, with a slight modification